src/HttpServer.ts

```diff
@@ -113,11 +136,56 @@ export class HttpServer {
         }
     }
 
-    statusHandler(req: any, res: any): void {
+    /**
+     * @openapi
+     * /status:
+     *   get:
+     *     summary: It returns a 204 response with an empty body.
+     *     description: It may be used just to check if the server is running.
+     *     responses:
+     *       '204':
+     *         description: No Content.
+     */
+    statusHandler(_: Request, res: Response): void {
         // TODO: check components and return proper status code
         res.status(204).end();
     }
 
+    /**
+     * @openapi
+     * /relay:
+     *   post:
+     *     summary: It relay transactions.
+     *     description: It receives transactions (deploy or forward transactions) and after performing all the checks it broadcast them to the `relayHub`.
+     *     requestBody:
+     *       description: Deploy transaction or forward transaction.
+     *       required: true
+     *       content:
+     *         application/json:
+     *           schema:
+     *             oneOf:
+     *               - $ref: '#/components/schemas/DeployTransactionRequest'
+     *               - $ref: '#/components/schemas/RelayTransactionRequest'
+     *           examples:
+     *             deploy:
+     *               summary: "Deploy request example"
+     *               value: {"relayRequest":{"request":{"relayHub":"0x<relay_hub>","to":"0x0000000000000000000000000000000000000000","data":"0x","from":"0x<from_address>","value":"0","nonce":"0","tokenAmount":"<token_amount>","tokenGas":"<token_gas>","tokenContract":"0x<token_contract>","recoverer":"0x0000000000000000000000000000000000000000","index":"1"},"relayData":{"gasPrice":"<gas_price>","callVerifier":"0x<call_verifier>","domainSeparator":"0x<domain_separator>","callForwarder":"0x<call_forwarder>","relayWorker":"0x<relay_worker>"}},"metadata":{"relayHubAddress":"0x<relay_hub>","signature":"0x<tx_signature>","relayMaxNonce":1}}
+     *             forward:
+     *               summary: "Forward request example"
+     *               value: {"relayRequest":{"request":{"relayHub":"0x<relay_hub>","to":"0x<to_address>","data":"0x<call_data>","from":"0x<from_address>","value":"0","nonce":"0","gas":"<gas>","tokenAmount":"<token_amount>","tokenGas":"<token_gas>","tokenContract":"0x<token_contract>"},"relayData":{"gasPrice":"<gas_price>","callVerifier":"0x<call_verifier>","domainSeparator":"0x<domain_separator>","callForwarder":"0x<call_forwarder>","relayWorker":"0x<relay_worker>"}},"metadata":{"relayHubAddress":"0x<relay_hub>","signature":"0x<tx_signature>","relayMaxNonce":4}}
```

Comment:
i would opt to have actual values here as examples, since (for instance) `0x<relay_worker>` doesn't really add info when the field is called `relayWorker`

Reply:
would you opt for arbitrary/invalid values?

Reply:
i think so, yes